ores/applications/celery.py

```diff
@@ -0,0 +1,58 @@
+"""
+Runs a celery worker.
+
+Usage:
+    celery [--config-dir=<path>]... [--logging-config=<path>]
+
+Options:
+    -h --help                Prints this documentation
+    --config-dir=<path>      The path to a directory containing configuration
+                             [default: config/]
+    --logging-config=<path>  The path to a logging configuration file
+"""
+import glob
+import logging
+import logging.config
+import os.path
+
+import docopt
+import yamlconf
+
+from ..score_processors import Celery
+
+DEFAULT_DIRS = ["config/", "/etc/ores/"]
+
+
+def main(argv=None):
+    args = docopt.docopt(__doc__, argv=argv)
+
+    run(config_dirs=args['--config-dir'],
+        logging_config=args['--logging-config'])
+
+
+def run(*args, **kwargs):
+    application = build(*args, **kwargs)
+    logging.getLogger('ores').setLevel(logging.DEBUG)
+    application.worker_main(argv=["celery_worker", "--loglevel=INFO"])
+
+
+def build(config_dirs=DEFAULT_DIRS, logging_config="logging_config.yaml"):
+    config_paths = []
+    for dir in config_dirs:
+        config_paths.extend(glob.glob(os.path.join(dir, "*.yaml")))
+    config_paths.sort()
```

Comment:
How is the naming of that yaml files supposed to work? If `config_paths` has `/a/b/c/99-foo.yaml` and `/z/y/x/00-bar.yaml`, would the loading be in the expected order?